ReactWindows/ReactNative.Shared/Bridge/ReactInstance.cs

```diff
@@ -33,6 +33,9 @@ class ReactInstance : IReactInstance
 
         private bool _initialized;
 
+        public event TransitionToBridgeIdleCallback OnTransitionToBridgeIdleCallback;
```

Comment:
>OnTransitionToBridgeIdleCallback [](start = 52, length = 32)

.NET events typically don't have an `On` prefix, and likely not a `Callback` suffix, since both of these are implied by the nature of the event. Can we just call these `BridgeIdle` and `BridgeBusy`?

Reply:
changed to an Action as per above and named according to other similar Actions in codebase.